client/js/src/client.ts

```diff
@@ -932,6 +933,12 @@ export function api_factory(
 										close_stream();
 									}
 								};
+								if (event_id in pending_stream_messages) {
```

Comment:
What will happen if `event_id` is not in `pending_stream_messages`? Will the client send another SSE message?

Reply:
pending_stream_messages is only the list of events that beat the race condition specified in the PR (event messages received before the POST request completed). In most cases there are no pending_stream_messages, in which case there is nothing to do.